collections/m2m gateway/certifiedAttributes/Create certified attribute.bru

```diff
@@ -29,3 +29,21 @@ script:pre-request {
   bru.setVar("randomName", `test name ${randomCode}`)
   bru.setVar("randomDescription", `test description ${randomCode}`)
 }
+
+docs {
+  Creates a new certified attribute.
+  
+  This endpoint allows the creation of certified attributes that can be officially recognized by the platform.
+  Only users with M2M_ADMIN role can create certified attributes.
+  
+  Request body should include:
+  - code: Unique identifier code for the attribute
+  - name: Display name of the attribute
+  - description: Detailed description of the attribute
+  
+  Returns:
+  - 201: Certified attribute created successfully
+  - 400: Invalid request body
+  - 401: Unauthorized
+  - 409: Attribute with same code already exists
+}
```

Comment:
These docs in the Bruno files are not strictly necessary: we already have the spec with its descriptions of various fields, error codes, etc.
Otherwise, these become one more thing to maintain, which will easily go out of sync with the actual implementation (same for other Bruno files in the PR)